e2e-tests/.ci/server.start.sh

```diff
@@ -2,89 +2,17 @@
 set -e -u -o pipefail
 cd "$(dirname "$0")"
 . .e2erc
-. .e2erc_setup
-
-if [ "$TEST" != "cypress" ] && [ "$TEST" != "playwright" ] && [ "$TEST" != "server" ]; then
-  mme2e_log "Invalid TEST='$TEST', expected: 'cypress', 'playwright' or 'server'"
-  exit 1
-fi
 
 BRANCH_DEFAULT=$(git branch --show-current)
 export BRANCH=${BRANCH:-$BRANCH_DEFAULT}
 BUILD_ID_DEFAULT=$(date +%s)
 export BUILD_ID=${BUILD_ID:-$BUILD_ID_DEFAULT}
 export CI_BASE_URL="${CI_BASE_URL:-localhost}"
-export SITE_URL="${SITE_URL:-http://server:8065}"
```

Comment:
Searching for `SITE_URL` within the `e2e-test/` directory doesn't yeld any result, so I thought this variable was actually not used anywhere and removed it. Is it actually used by any code?

Reply:
Oh, you're right, not there and also I missed :). It's meant to override - https://github.com/mattermost/mattermost/blob/master/e2e-tests/.ci/docker_compose_generator.sh#L195. So I guess, it's fine to remove for now or you might want to include. I'm fine either way.

Reply:
We're now using the host's network, instead of the container network, so we shouldn't have a need to customize the site URL anymore since it's `localhost` all the time :smile: